raiden_contracts/tests/test_user_deposit_contract.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from eth_tester.exceptions import TransactionFailed
+
+from raiden_contracts.constants import CONTRACTS_VERSION, UserDepositEvent
+
+
+def test_deposit(
+    user_deposit_contract,
+    custom_token,
+    get_accounts,
+):
+    (A, B) = get_accounts(2)
+    custom_token.functions.mint(30).transact({'from': A})
+    custom_token.functions.approve(user_deposit_contract.address, 30).transact({'from': A})
+
+    # deposit to A's own balance
+    user_deposit_contract.functions.deposit(A, 10).transact({'from': A})
+    assert user_deposit_contract.functions.balances(A).call() == 10
+    assert custom_token.functions.balanceOf(A).call() == 20
```

Comment:
Also check `custom_token.functions.balanceOf(user_deposit_contract.address)`.

Reply:
Good point! Done in f07f76d.